vcfa/datasource_vcfa_kube_config.go

```diff
@@ -0,0 +1,157 @@
+package vcfa
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/golang-jwt/jwt/v5"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/go-vcloud-director/v3/ccitypes"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
```

Comment:
Can you please import `clientcmdapi "k8s.io/client-go/tools/clientcmd/api/v1` instead, the one without the `v1` is an old/deprecated one that should not be used (that was wrong in my original change). You may have to adjust a bit the below `clientcmdapi.Config`, as the new versions do not use maps anymore (they use arrays now).

Reply:
Changed.